src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/BaseRecalibratorWorker.java

```diff
@@ -242,8 +242,9 @@ private ReadTransformer makeReadTransform() {
      * The "knownLocations" list doesn't need to be complete but it must includes all those that overlap with the read.
      */
     public void apply(GATKRead originalRead, final ReferenceContext ref, List<? extends Locatable> knownLocations) {
+        final GATKRead copy = originalRead.copy();  // REMOVE ME?
```

Comment:
Opps! It was there to help me keep track of this that _might_ be the cause. It  was the cause (so I'm removing the comment).